scripts/generate_skip_file.py

```diff
@@ -0,0 +1,28 @@
+#%%
+import re
+
+with open("data.txt", "r") as f:
+    data = f.read()
+
+matches_failed = re.findall(r"thread '(.*)' panicked at ", data)
+matches_failed = matches_failed[1:]
+
+print(len(matches_failed))
+
+print(matches_failed[:10])
+
+matches_failed = [(m.split('::')[-2].replace("_minus_", "-"), m.split('::')[-1]) for m in matches_failed]
```

Comment:
I added 
```
    /// Formats the given string into a valid rust identifier.
    fn format_into_identifier(s: &str) -> String {
        s.replace('-', "_minus_")
            .replace('+', "_plus_")
            .replace('^', "_xor_")
    }
```
could you update this line to match

Reply:
fixed.